content/source/docs/enterprise/sentinel/import/index.html.md

```diff
@@ -17,6 +17,18 @@ state and plan.
 - [tfconfig](./tfconfig.html) - This provides access to a Terraform configuration, the set of "tf" files that are used to describe the desired infrastructure state.
 - [tfstate](./tfstate.html) - This provides access to the Terraform state, the file used by Terraform to map real world resources to your configuration.
 
+To create test data for rules, Terraform Enterprise allows you to create mocks
+of these imports from executable plans for use with the mocking or testing
+features of the [Sentinel Simulator](https://docs.hashicorp.com/sentinel/commands/).
```

Comment:
This sentence kind of says "To X" twice. I wonder if it makes sense to remove the first one.